gulp/js/views/transcripts/edit.js

```diff
@@ -20,6 +20,10 @@ app.views.TranscriptEdit = app.views.Transcript.extend({
     $(window).trigger('scroll-to', [$('#completion-content'), 100]);
   },
 
+  showConventions: function(){
+    this.$('.conventions-page').toggleClass( "active"  )
```

Comment:
I find `toggleClass` to be a bit fickle, not working 100% as expected. Have you tested it a lot, doing really random things on the page?
I find having conditions and add and remove classes to be more robust is all

Reply:
@Rhiana , I tested locally and seems like working. But not to the extend with doing all sort of random things. 

I'm thinking to leave this for now and see if we see and issues down the line. Because this section is not something heavily used. This is most for the new users to understand the convenntions 

any thoughts ?